packages/admin-ui-codemod/src/space-tokens-review.js

```diff
@@ -0,0 +1,139 @@
+const spacingProps = {
```

Comment:
what about the first tokens that we have? (`1,2,3,4,5,6,7,8, px, 2px`) Shouldn't we replace them too?